### PR TITLE
chore: Add Windows 10c image as default (August optional patches) 

### PR DESCRIPTION
**Pull request by jsturtevant**
**Reason for Change**:

**Issue Fixed**:

**Credit Where Due**:

Does this change contain code from or inspired by another project?

- [X] No
- [ ] Yes

If "Yes," did you notify that project's maintainers and provide attribution?

- [ ] No
- [ ] Yes

**Requirements**:

- [X] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [X] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: pkg/api/azenvtypes.go [hunk 1]
```diff
@@ -186,9 +186,9 @@ var (
 	// AKSWindowsServer2019OSImageConfig is the AKS image based on Windows Server 2019
 	AKSWindowsServer2019OSImageConfig = AzureOSImageConfig{
 		ImageOffer:     "aks-windows",
-		ImageSku:       "2019-datacenter-core-smalldisk-2010",
+		ImageSku:       "2019-datacenter-core-smalldisk-2011",
 		ImagePublisher: "microsoft-aks",
-		ImageVersion:   "17763.1397.201006",
+		ImageVersion:   "17763.1554.201109",
```
Comment (marosset) [inline pkg/api/azenvtypes.go:191]: Just to confirm we want to keep the WindowsServer2019OSImageConfig values below on the Aug patches right?
I think we do and then we can update that with 11B (which should have all the fixes also present in 10C)

**Review by marosset — APPROVED** [review]
/lgtm

**Review by jackfrancis — APPROVED** [review]
/lgtm

Thank you @jsturtevant getting here was a lot of work!!!

File: pkg/api/azenvtypes.go (hunk 1 above)
Comment (jsturtevant) [reply pkg/api/azenvtypes.go:191]: yes.  Since there is no base image built for 10c and 9b/10b have networking issues.

**Pull request merged**